boards/arm/raytac_mdbt50q_db_40_nrf52840/raytac_mdbt50q_db_40_nrf52840.dts

```diff
@@ -0,0 +1,199 @@
+// Copyright (c) 2022 Raytac Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+#include "raytac_mdbt50q_db_40_nrf52840-pinctrl.dtsi"
+
+/ {
+	model = "Raytac MDBT50Q-DB-40_nRF52840";
+	compatible = "raytac,raytac-mdbt50q-db-40-nrf52840";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,bt-mon-uart = &uart0;
+		zephyr,bt-c2h-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
```

Comment:
As above, `zephyr,ieee802154 = &ieee802154;`

Reply:
I will add it in the next commit.